network-api/networkapi/buyersguide/templates/fragments/it_uses.html

```diff
@@ -0,0 +1,17 @@
+{% if device != False or app != False %}
```

Comment:
(asking because I don't have much context for PNI changes) What is `app`?

Reply:
some products also come with an app that you run on your phone, in which case the phone's camera/mic/location might get used